robot_poses:
  length: 4
  "0": [0,0.8,0.8,0,0.8,0]
  "1": [0,0.6,0.8,0,0.8,0]
  "2": [2.8,0.6,0.8,0,0.8,0]
  "3": [2.8,0.8,0.8,0,0.8,0]
robot_sequence:
  length: 8
  "0":
    type: 0
    goal: 1
  "1":
    type: 0
    goal: 0
  "2":
    type: 1
  "3":
    type: 0
    goal: 1
  "4":
    type: 0
    goal: 2
  "5":
    type: 2
  "6":
    type: 0
    goal: 3
  "7":
    type: 0
    goal: 2
human_sequence:
  length: 3
  start_pose: []
  "0":
    reach_target: [-0.35,0.1,0.05]
    end_delay: 2.0
  "1":
    reach_target: [-0.35,0.1,0.05]
    end_delay: 2.0
  "2":
    reach_target: [-0.35,0.1,0.05]
    end_delay: 2.0
